package/src/components/Channel/Channel.tsx

```diff
@@ -672,6 +673,30 @@ const ChannelWithContext = <
     }
   };
 
+  const handleAppBackground = useCallback(
```

Comment:
why not just defined it inside `useEffect`? And add `thread?.id` and `channelId` to useEffect's dependency

Reply:
Actually can we reuse `useAppStateListener` somehow? What do you think?